cdc/tests/integration_tests/autorandom/run.sh

```diff
@@ -9,14 +9,16 @@ CDC_BINARY=tikv-cdc.test
 SINK_TYPE=$1
 UP_PD=http://$UP_PD_HOST_1:$UP_PD_PORT_1
 DOWN_PD=http://$DOWN_PD_HOST:$DOWN_PD_PORT
+# fallback 10s
+FALL_BACK=2621440000
 
 function run() {
 	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
 	start_tidb_cluster --workdir $WORK_DIR
 	cd $WORK_DIR
 
 	start_ts=$(tikv-cdc cli tso query --pd=$UP_PD)
-	sleep 10
+	start_ts=$(expr $start_ts - $FALL_BACK)
```

Comment:
The new commits LGTM.

Nit: extract as a utility function.

Reply:
Yesh, I extracted as a utility function. PTAL~